routes/register.js

```diff
@@ -11,10 +11,37 @@ const hashPassword = function(password) {
 router.post('/', function(req, res) {
   const { body } = req;
   if(!body.username || !body.password || !body.name || !body.phone)
-  {
+  { console.log("Invalid body");
     res.send(400,"Invalid Body");
     return;
   }
+  //console.log(typeof body.phone);
+  if(body.phone.length!=10 || isNaN(Number(body.phone))){
+    //console.log(body.phone.length!=10)
+    //console.log(isNaN(Number(body.phone)))
+    console.log("Invalid phone");
+    res.send(400,"Phone number not valid");
+    return;
+  }
+  //console.log(!validator.isAlphanumeric(body.username));
+  if(!validator.isAlphanumeric(body.username)){
```

Comment:
Seems you are using a middleware called validator. Please commit the package.json file and app.js files if there are any changes

Reply:
Added validator to package.js.